link-verifier/verify-links.py

```diff
@@ -254,7 +258,10 @@ def fetch_issues(repo, issue_type, limit):
         if process.returncode == 0:
             key = issue_type + 's'
             for issue in process.stdout.split():
-                main_repo_list[repo][key].add(int(issue))
+                if(issue.isnumeric()):
```

Comment:
If for some reason the GitHub issues can't be accessed this will throw an error attempting to convert it to an int

Check if we have an actual number first, if we do not that means there was an error reading the actual Issues from the repo. When this occurs it returns an output of: